mock_drivers/rosbag_mock_drivers/CMakeLists.txt

```diff
@@ -0,0 +1,149 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(rosbag_mock_drivers)
+
+## Compile as C++14
+add_compile_options(-std=c++14)
+
+
+####################
+## Dependancies   ##
+####################
+set ( PKG_CATKIN_DEPENDS 
+  roscpp
+  std_msgs
+  carma_utils
+  cav_msgs
+  cav_srvs
+  rosbag
```

Comment:
Do we need to specify this? Usually we were able to use it directly.

Reply:
Nope. Good catch, this is left over from an earlier version that used the rosbag API.